meson.build

```diff
@@ -4,7 +4,7 @@ project('classdesc', 'cpp',
     meson_version: '>=0.39.1',
     license: [ 'GPL3+', 'MIT' ],
     default_options: [
-        'cpp_std=c++14',
+        'cpp_std=c++11',
```

Comment:
For reasons I may have alluded to, the main part of classdesc needs to be built with the C++98 standard with TR1. Travis needs to test this. There is one test directory for handling modern C++11 (and beyond) features.
I expect in the next few years to relax the C++98 requirement, probably to the C++11.